### DISCUSSION TITLE
Version overlay display conflicts with open game overlays

### DISCUSSION THREAD

**Discussion by frenzibyte** [ppy/osu#16064] [Q&A] [answered]
Split from #16040 

One way to fix this would be to make the version overlay ingrained with the `MainMenu` screen and behind the overlay layer.

https://user-images.githubusercontent.com/22781491/145746151-7ab380cb-81f1-4874-bc03-6ba2ff94b0ad.mp4

I think that looks fine to me, unless there are envisioned plans for the look of the `VersionManager` that makes this change incorrect.

The part about the `VersionManager` scaling with the `MainMenu` screen is not necessary, but I'm not sure if it looks good the original way either.

**Comment by peppy** [comment]
Moving it to the main menu is fine.

**Comment by frenzibyte** [comment, marked as answer]
Resolved in #16065
